AlphaWallet/UI/Views/AmountTextField.swift

```diff
@@ -11,6 +11,7 @@ class AmountTextField: UIControl {
     enum Currency: String {
         case eth = "ETH"
         case usd = "USD"
+        case xDai = "xDAI"
```

Comment:
Think it should be "DAI" instead of "xDAI"?

Reply:
Another way to do this is to change `Currency` to:

```
    enum Currency {
        case cryptoCurrency(String)
        case usd(String)
    }
```

But this implies that only the display value is different ("DAI" vs "ETH"), but handling is the same. If that's valid, this might be cleaner.

Reply:
@hboon yes it functions the same way as a native currency, putting this in.

Reply:
> @hboon not DAI, DAI is on the mainchain, since the xdai chain runs on POA, the native currency is xDAI rather than regular DAI. I make the distinction clear to the user.

Ah. I see. Thanks.